styles/colors.css

```diff
@@ -0,0 +1,12 @@
+:root{
+
+    /*background colors*/
+    --clr-blue:#B9eAed;

```

Comment:
Naming of the color variables could be better.

Reply:
Sure will keep in mind and make changes.